tests/unit/test_torch_dataloader.py

```diff
@@ -39,6 +39,53 @@
 GPU_DEVICE_IDS = [d.id for d in numba.cuda.gpus]
 
 
+@pytest.mark.parametrize("batch_size", [10, 9, 8])
+@pytest.mark.parametrize("drop_last", [True, False])
+@pytest.mark.parametrize("num_rows", [100])
+def test_tf_drp_reset(tmpdir, batch_size, drop_last, num_rows):
```

Comment:
Can we go with a more descriptive name for the test?  I assume that 'tf' here means tensorflow - maybe we should rename this to torch here?  Also I'd prefer 'drop' to 'drp' 

```suggestion
def test_torch_drop_reset(tmpdir, batch_size, drop_last, num_rows):
```

(also this test seems to be failing in CI - and I'm not sure why =(

Reply:
yeah its because of the difference in external facing batch tuple format. I am working on it now. But will change name.